### PR TITLE
Upgrade cucumber

### PR DESCRIPTION
**Pull request by patricksmms**
This transformer is using an outdated version of cucumber. Upgrading here to the latest, now moved into `@cucumber/cucumber`.

### CODE REVIEW

File: src/parsers/suite.ts [hunk 1]
```diff
@@ -10,12 +12,22 @@ import {parseSteps} from './steps';
...
 import {matchesTags} from './tags';
 
+interface Spec {
+    document: messages.GherkinDocument.IFeature;
+    afterEach: TestCaseHookDefinition[];
+    afterAll: TestCaseHookDefinition[];
+    beforeEach: TestCaseHookDefinition[];
+    beforeAll: TestCaseHookDefinition[];
+    skip: Boolean;
+    suites: any;
```
Comment (patricksmms) [inline src/parsers/suite.ts:22]: I didn't dig enough to get the type of `suites`, but it's worth getting it right.

**Comment by mainfraame** [comment]
Thanks for contributing @patricksmms. I don't see an issue merging this. I'll publish the 0.1.4 version w/ this later on tonight.

**Comment by patricksmms** [comment]
Thank you!

**Comment by mainfraame** [comment]
@patricksmms I'm going to make this a minor version increase, instead of a patch (0.2.0) since we are upgrading the cucumber version. I'll also update the README with the newer cucumber import

**Comment by mainfraame** [comment]
@patricksmms 0.2.0 is published. Thank you again for contributing...

**Pull request merged**